docs/source/tutorials/playing-command.rst

```diff
@@ -102,24 +123,24 @@ And so, in chat, you will see that::
 
     <YourNick> .hello
     <Sopel> Hello YourNick, have a nice day!
-    *YourNick hi*
+    * YourNick hi
```

Comment:
This is how IRCCloud displays the result of a `/me` command:

![image](https://github.com/sopel-irc/sopel/assets/3259552/b02abef7-0b63-46c8-9a57-34141d6a2230)

It doesn't add "says" or anything, because in my experience it is supposed to be a user's action, and not a user talking.

Reply:
```suggestion
    * YourNick waves
```